### DISCUSSION TITLE
Unable to convert this issue to a discussion

### DISCUSSION THREAD

**Discussion by dounokouno** [community/community#2924] [Discussions]
The below issue could not be converted to discussion.

https://github.com/dounokouno/TransmitMail/issues/2

What kind of problem can you think of? Also, is there anything I can do as a repository administrator?

Thank you.

**Comment by becca** [comment]
What error are you receiving when you are trying to convert? Please provide as much information as you can.

**Reply by dounokouno** [reply]
Thank you for your comment.

I received the error message "Unable to convert this issue to a discussion."

<img width="1437" alt="スクリーンショット 2020-12-11 11 04 20" src="https://user-images.githubusercontent.com/215381/101852344-c6243580-3ba0-11eb-92f9-0a8c7ca1eec8.png">

**Comment by Borda** [comment]
It seems that now even the option of converting "issue to discussion" is missing, is it on purpose?

**Reply by dounokouno** [reply]
That one, it's true. I tried converting to discussion on the corresponding issue a few days ago as well. I wonder if the feature has been disabled because the link to convert from issue to discussion is missing for all but the issue in question...

**Reply by Borda** [reply]
according to the [docs](https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion), it shall be possible, so it is a bug?

**Reply by dounokouno** [reply]
Hmm, I don't know...

**Reply by erikr** [reply]
Others are seeing the same problem https://github.com/github/feedback/discussions/3088

**Comment by becca** [comment]
@Borda @erikr and anyone else following along: 

we have recently disabled the ability to convert issues into discussions because of a few bugs that folks have made us aware of. we're working to resolve this issue and will be re-enabling conversions as soon as all of the underlying issues (no pun intended) are resolved. i'll keep you posted on when we re-enable conversions.

**Reply by Borda** [reply]
cool, do you have any ETA, if we are talking about days or rather weeks?

**Reply by becca** [reply]
We're working to resolve this ASAP. It's a top priority for us. I'll keep you posted.

**Reply by erikr** [reply]
@becca Thanks so much for clarifying! Looking forward to the feature being re-enabled. Also, we appreciate your diligent engagement with the users here.

**Reply by patrickhulce** [reply]
Are there any updates on the progress here (in terms of the "is it possible this is a weeks-long problem" question)? We are also very much looking forward to the feature being re-enabled :)

**Reply by Harmon758** [reply]
See https://github.com/github/feedback/discussions/2952 and https://github.com/github/feedback/discussions/3105.

**Reply by sethvoltz** [reply]
It would be helpful if this was mentioned in the documentation for the feature. I spent a long time looking for the feature and searching around until I found this thread.

**Reply by Shaquu** [reply]
What is current progress on it?

**Comment by Koenkk** [comment]
I also cannot convert issues into discussion anymore. I've converted multiple using the "Convert this issue to discussion" button but it's missing now (?). If I remember correctly these were part of the button in the bottom but I can't find it anymore.

![image](https://user-images.githubusercontent.com/2892853/103446956-71e31180-4c85-11eb-9804-0d08d75e5144.png)

**Reply by Koenkk** [reply]
Never-mind, missed https://github.com/github/feedback/discussions/2924#discussioncomment-254592

Off topic: I really like Github discussions ❤️

**Comment by dounokouno** [comment]
Today, I tried converting from issue to discussion and it worked fine. Thank you for fixing the problem! 🙌

- Original issue: dounokouno/TransmitMail/issues/2
- Converted discussion: dounokouno/TransmitMail/discussions/72

**Comment by driesvints** [comment]
Seems to be available now! 🎉 

![Screen Shot 2021-07-01 at 09 25 41](https://user-images.githubusercontent.com/594614/124083698-7a5ab000-da4e-11eb-95c6-40c68062eb92.jpg)

**Comment by intelliot** [comment]
~~How can issues be converted in bulk?~~

Nevermind - found it here: https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#converting-issues-based-on-labels
